Setup/creating_test_namespace/runtest.sh

```diff
@@ -62,6 +62,7 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "Controller runs appropriately"
+        rlRun "sleep 10" 0 "Previous pods could be in terminating state and could be greped when function search fo name of pod."
```

Comment:
Maybe this "10" could be referenced as a constant, something like TO_WAIT_FOR_CONTROLLER_RUN (or the name you wish)

Reply:
greped -> grepped